### ISSUE TITLE
[Bug] Question mark breaks download

### ISSUE THREAD

**Issue by ardittristan** [Jays2Kings/tachiyomiJ2K#61]
### Device information
* Tachiyomi version: 0.9.69
* Android version: 8.0

## Steps to reproduce
Download a manga from "Nani? Scans"

### Expected behavior
It should download

### Actual behavior
It errors and gives invalid download location.

### Other details
I somehow got it to create a folder once, and it was called "nani_ scans" 

Download worked on tachiyomi dev

**Comment by Jays2Kings** [comment]
That source is able to work fine for me, the folder creation hasn't changed from dev. I would make it change the ? to a _ but that might break others downloads so I'd like to confirm first

**Comment by ardittristan** [comment]
I did notice that you can't create folders on my phone with a ? At all

**Comment by ardittristan** [comment]
Oh, and I'm talking about the nani scan source from manga cards, not from the mangadex extension

**Comment by Jays2Kings** [comment]
Closing since this is old, please reopen if the issue still exists

**Comment by Jerrk** [comment]
I still have this issue. it's created 30+ folders all called "Nani_ Scans (##)" in my downloads directory, and it stalled all of my downloads.
Removing the folder and removing the pending download from nani? scans fixed the problem.

So theres definitely still an issue with the ? in their name.

**Comment by Schadows** [comment]
Same here.

When a new chapter from a manga from "Nani? Scans" popped up in the download list, it blocked the whole list even when the incriminated chapter was moved to the bottom of the list.

I've checked the "Android/data/eu.kanade.tachiyomi.j2k/files" folder, but there seems to be no folder related to "Nani" in any way, which should explain why I get the "Invalid download location", despite the source works perfectly when reading the chapters without downloading them.

Don't know if the folder name is defined by the source extension itself, or simply the name of the extension, but whatever the case, prohibited characters in folder names should be prevented here.

Switched to another sources in the meantime.
